### PR TITLE
Use a single workspace for flashinfer

### PR DESCRIPTION
**Pull request by merrymercy**
with flashinfer >= 0.1.5, we can use a single workspace buffer for all wrappers.

### CODE REVIEW

**Comment by zhyncs** [comment]
ref https://github.com/flashinfer-ai/flashinfer/actions/runs/10385309831

File: python/sglang/global_config.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __init__(self):
         # Runtime constants: others
         self.num_continue_decode_steps = 10
         self.retract_decode_steps = 20
-        self.flashinfer_workspace_size = 192 * 1024 * 1024
+        self.flashinfer_workspace_size = 256 * 1024 * 1024
```
Comment (merrymercy) [inline python/sglang/global_config.py]:
```suggestion
        self.flashinfer_workspace_size = 320 * 1024 * 1024
```

File: python/sglang/global_config.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def __init__(self):
         # Runtime constants: others
         self.num_continue_decode_steps = 10
         self.retract_decode_steps = 20
-        self.flashinfer_workspace_size = 192 * 1024 * 1024
+        self.flashinfer_workspace_size = 384 * 1024 * 1024
```
Comment (hnyls2002) [inline python/sglang/global_config.py:30]: Yeah, we truly need this large work space... otherwise the chunked prefill would probably crash.

**Comment by zhyncs** [comment]
Why the UT failed?

**Pull request merged**